torch/csrc/autograd/variable.cpp

```diff
@@ -109,6 +111,22 @@ void Variable::Impl::detach_() {
   output_nr_ = 0;
 }
 
+void Variable::Impl::backward(
```

Comment:
nit: Can we give the C++ API the same argument set as this one?

Reply:
What do you mean? I've removed the C++ API one, so you'd only use this method

Reply:
Ah my bad, I missed it. LGTM